### PR TITLE
MangaSwat: fix search

### LINKED ISSUE #17238: MangaSwat search and last update not working
### Source information

MangaSwat 1.4.36

### Source language

Arabic 

### Steps to reproduce

1.if i tried to search for a manga . it doesn't work out. just showing a random mangas
2. the same happened if i uesd latest just showing a random mangas

### Expected behavior

it should work like this 
Screenshot_2023-07-23-23-06-28-060_eu kanade tachiyomi j2k

### Actual behavior

but this happened instead 

### Tachiyomi version

tachiyomij2k 1.6.6

### Android version

Android 11

### Other details

my library working and updating fine no problem with it

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by vetleledaal**
Closes #17238

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/overrides/mangathemesia/mangaswat/src/MangaSwat.kt [hunk 1]
```diff
@@ -41,6 +43,21 @@ class MangaSwat : MangaThemesia(
     override fun headersBuilder(): Headers.Builder = super.headersBuilder()
         .add("Referer", "$baseUrl/")
 
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val request = super.searchMangaRequest(page, query, filters)
+        val urlBuilder = request.url.newBuilder()
+
+        if (query != "") {
```
Comment (alessandrojean) [inline multisrc/overrides/mangathemesia/mangaswat/src/MangaSwat.kt:50]: A more Kotlin-ish version.

```suggestion
        if (query.isNotEmpty()) {
```

**Comment by T3MYA11** [comment]
okay now the problem still exist and also I can't update the manga In my library it doesn't give me error or anything just not updating also latset and mangal list just showing a 25 manga and most of them not even have new chapters in the website @vetleledaal

File: multisrc/overrides/mangathemesia/mangaswat/src/MangaSwat.kt (hunk 1 above)
Comment (AntsyLich) [reply multisrc/overrides/mangathemesia/mangaswat/src/MangaSwat.kt]: Why not `isNotBlank`

Comment (vetleledaal) [reply multisrc/overrides/mangathemesia/mangaswat/src/MangaSwat.kt]: Depends on how the site handles whitespace, will check this evening (UTC)

**Comment by vetleledaal** [comment]
> okay now the problem still exist

The changes are not merged yet. Wait for the changes to be approved, and it should be available in the app shortly after. I have also addressed the 25 manga limit.

**Pull request merged**